conf/dhcpd.conf

```diff
@@ -10,3 +10,9 @@ log-facility local6;
 %%active%%
 
 %%networks%%
+
+group parking {
```

Comment:
Shouldn't we add a little small comment to say something like "httpd.portal parking related" or whatnot just so that we remember in 30 years that is what not a "Stationnement Montreal" feature related